src/mercury_bulk.h

```diff
@@ -18,12 +18,6 @@
 /* Public Macros */
 /*****************/
 
-/* The memory attributes associated with the bulk handle
- * can be defined as read only, write only or read-write */
-#define HG_BULK_READ_ONLY  (1 << 0)
-#define HG_BULK_WRITE_ONLY (1 << 1)
-#define HG_BULK_READWRITE  (HG_BULK_READ_ONLY | HG_BULK_WRITE_ONLY)
-
```

Comment:
please keep those like that, we need to only have those public, the rest must remain private.